src/Microsoft.Azure.WebJobs.Host/Bindings/StorageAccount/CloudStorageAccountBindingProvider.cs

```diff
@@ -28,14 +28,15 @@ public CloudStorageAccountBindingProvider(IStorageAccountProvider accountProvide
         {
             ParameterInfo parameter = context.Parameter;
 
-            if (context.Parameter.ParameterType != typeof(CloudStorageAccount))
+            if (parameter.ParameterType != typeof(CloudStorageAccount))
             {
                 return null;
             }
 
-            IStorageAccount account = await _accountProvider.GetStorageAccountAsync(context.Parameter, context.CancellationToken);
```

Comment:
Reworked this so I could remove the ParameterInfo based overload - this was the last usage.